packages/kbn-es/src/cluster.js

```diff
@@ -238,6 +238,10 @@ exports.Cluster = class Cluster {
 
     this._process = execa(ES_BIN, args, {
       cwd: installPath,
+      env: {
+        ...process.env,
+        ...(options.esEnvVars || {}),
```

Comment:
__question:__ @elastic/kibana-operations how do you feel about this change? We need to specify some JVM options for the test ES instance (namely `-Djava.security.krb5.conf` and `-Dsun.security.krb5.rcache`) and I'm aware of only two ways of doing that, either via `config/jvm.options` or `ES_JAVA_OPTS` env variable.

Dealing with `ES_JAVA_OPTS` seemed easier to me, so I picked that option, but happy to hear about any better ways of doing that. With this change test config would look like this:
```ts
{
    esTestCluster: {
      ...xPackAPITestsConfig.get('esTestCluster'),
      serverArgs: [
        ...xPackAPITestsConfig.get('esTestCluster.serverArgs'),
        `xpack.security.authc.realms.kerberos.kerb1.keytab.path=${kerberosKeytabPath}`,
      ],
      serverEnvVars: {
        ES_JAVA_OPTS: `-Djava.security.krb5.conf=${kerberosConfigPath} -Dsun.security.krb5.rcache=none`,
      },
    },
    ...
  };
```

Reply:
Works for me, though I think you might run into problems with paths containing spaces based on your snippet.

Reply:
Great!

> I think you might run into problems with paths containing spaces based on your snippet.

Thanks for pointing out! We shouldn't have spaces there, but yeah, I pretend this problem doesn't exist yet :see_no_evil: